nixos/tests/all-tests.nix

```diff
@@ -306,7 +306,7 @@ in
   cinnamon-wayland = runTest ./cinnamon-wayland.nix;
   cjdns = runTest ./cjdns.nix;
   clatd = runTest ./clatd.nix;
-  clickhouse = runTest ./clickhouse.nix;
+  clickhouse = handleTest ./clickhouse { };
```

Comment:
Please use `runTest` instead, see https://github.com/NixOS/nixpkgs/issues/386873
Take a look at `nginx-variants` test for an example of a nested tests which use `runTest`